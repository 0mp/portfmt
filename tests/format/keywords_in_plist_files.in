PLIST_FILES=	"@sample etc/foobar.sample" \
		bin/asdfasd
