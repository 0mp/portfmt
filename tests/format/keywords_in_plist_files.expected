PLIST_FILES=	bin/asdfasd \
		"@sample etc/foobar.sample"
